I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 28, in script call
    call story #story is the label.
  File "game/story.rpy", line 931, in script
    jump choice_Sayori2
ScriptError: could not find label 'choice_Sayori2'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 28, in script call
    call story #story is the label.
  File "game/story.rpy", line 931, in script
    jump choice_Sayori2
  File "renpy/ast.py", line 1757, in execute
    rv = renpy.game.script.lookup(target)
  File "renpy/script.py", line 918, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'choice_Sayori2'.

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
Doki Doki Detective Club 1.1.2
Sat Aug 17 21:00:51 2024
